lib/cc/cli/analyze.rb

```diff
@@ -43,7 +43,7 @@ def registry
       end
 
       def formatter
-        @formatter ||= Formatters::PlainTextFormatter.new
+        @formatter ||= Formatters::PlainTextFormatter.new(filesystem)
```

Comment:
We need to adjust the usage of `Formatters.resolve` so it also passes a `filesystem` param